lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -183,8 +193,66 @@ class RenderForm extends React.Component {
     );
   }
 
+  async handleInputChange(event) {
+    event.preventDefault();
+    const queryString = event.target.value;
+    if (queryString.length > 2) {
+      let suggestions = [];
+      await fetch(`${this.learnerSupportCenterURL}/hc/api/internal/instant_search.json?query=${queryString}&locale=en-us`, { headers: {
```

Comment:
are you validating queryString for any XSS checks?

Reply:
I haven't add any checks. Let me check if the input is exploitable to XSS attacks. 👍

Reply:
It is indeed XSS safe.